config/initializers/metadecidim_menu.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+Decidim.menu :metadecidim_menu do |menu|
+  menu.remove_item :assemblies
```

Comment:
I actually think you don't have to remove the items for the `:metadecidim_menu` because it's not defined in the core.

Reply:
I agree, they don't make sense anymore.  Done in 62dc22b90fd0e844229b796a3642874a38cf9f5d